pkg/registry/apis/provisioning/register.go

```diff
@@ -106,7 +107,8 @@ type APIBuilder struct {
 	legacyMigrator    legacy.LegacyMigrator
 	storageStatus     dualwrite.Service
 	unified           resource.ResourceClient
-	repositorySecrets secrets.RepositorySecrets
+	decryptSvc        secret.DecryptService
```

Comment:
adding in this PR so we do not require an enterprise wire update in https://github.com/grafana/grafana/pull/107803